src/components/Main/Results/ResultsCard.tsx

```diff
@@ -131,7 +131,7 @@ function ResultsCardFunction({
               <LinkButton
                 className="new-tab-button"
                 color="secondary"
-                disabled={!scenarioUrl}
+                disabled={!canRun}
```

Comment:
Could it be that we canRun but the scenarioURL is not ready?

Reply:
Good question. The app will immediately try to run the scenario provided in a URL link, so it makes sense to ensure that it is indeed runnable before enabling the run in new tab.